kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -607,4 +607,9 @@
     line-height: 2em;
   }
 
+  .small {
+    max-width: 100%;
+    overflow-x: hidden !important;
+  }
```

Comment:
This change strikes me as a bit risky. `!important` has a tendency to cause unexpected breakage and going on `CoreBase` the effects could be wide-ranging.

Reply:
I actually am pretty sure this is not needed and was something I was using for troubleshooting. Good catch! I'll definitely remove it, and will update the PR to indicate whether it changes any of the items described above, but I'm hoping it won't.

Reply:
Manual QA seems to still check out with this removed